src/main/target/STM32_UNIFIED/target.h

```diff
@@ -78,6 +78,33 @@
 #define TARGET_IO_PORTD 0xffff
 #define TARGET_IO_PORTE 0xffff
 
+#elif defined(STM32F411_BMI270)
+#define TARGET_BOARD_IDENTIFIER "411B"
```

Comment:
Can we keep the structure consistent: `S41B`

Reply:
will change